conda_libmamba_solver/solver.py

```diff
@@ -640,6 +640,9 @@ def _package_info_to_package_record(
         # Signature verification requires channel information _with_ subdir data
         channel = Channel(pkg.channel)
         if not channel.subdir:
+            # conda caches channels created with a call, these should not be mutated.
+            # Create a new channel that is not cached and mutate
+            channel = Channel.from_value(pkg.channel)
```

Comment:
This wont guarantee a copy, the caching happens in the metaclass and `Channel.from_value` still invokes the metaclass

Reply:
I was thinking `Channel.from_channel_name` but that also hit the cache.

Reply:
I hope we don't have to do something like `Channel._cache_.setdefault(...)` 😂

Reply:
`copy.deepcopy` will makes copy that is not in the cache.

Another option would be:
```
channel = Channel(**channel.dump()))
```

Reply:
Decided to use keyword arguments when creating the platform containing channel instance. This avoids the cache.